registration/TOS.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
```

Comment:
[optional] Please indent lines 3 - 19.
You want to nest head and body tags within html tag.